core/storage/skill/gae_models.py

```diff
@@ -254,6 +254,21 @@ def get_by_description(cls, description: str) -> Optional[SkillModel]:
         """
         return cls.get_all().filter(cls.description == description).get()
 
+    @classmethod
+    def get_by_prerequisite(cls, skillid: str) -> Sequence[SkillModel]:
+        """Gets SkillModels that contain the prerequisite defined by
+        skillid. Returns None if none exist.
+
+        Args:
+            skillid: str. The prerequisite's skillid.
+
+        Returns:
+            SkillModel|None. The skill models of the skill or None if not
```

Comment:
Don't know what the docstring here means. It talks about skill models (plural) but that doesn't match the type.

or None if not found --> or None if no models have the given skill as a prerequisite.